packages/m365_defender/changelog.yml

```
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "2.18.0"
```

```suggestion
- version: "2.17.1"
```
We use semver, so bugfixes are patch-bumping.